### ISSUE TITLE
????????? Kobra 2 1st model

### ISSUE THREAD

**Issue by marcokitt2000** [ANYCUBIC-3D/Kobra3#1]
Stop stop and make a new firmware for the kobra 2 1s model marlin based 3.06 here on github is not compleet after compile it it doesn,t auto z offset.
please fix it or make a firmware where the printer is working correct and m600 and z offset steps on the screen not 0.05mm but 0.01 mm or lower.
After the other kobra 2 series (neo mmmm) kobra 2 pro,plus and max than 1st release is kobra works great but need software correction or update to the latest marlin build etc...
Let this printer be the 1st kobra as a beast.

**Comment by germeshausen** [comment]
You should have a look at [Kobra 2](https://github.com/ANYCUBIC-3D/Kobra2) as this here is intended for Kobra 3 related topics.

**Comment by marcokitt2000** [comment]
I know that.

Why i do that Anycubic doesn do any thing. 

 I have now test it with klipper works much better now. M600 etc  and faster. If they asnwer and on my mail nothing we wait to much. If they upload 100% the source someone can add or make it work etc.

**Comment by YouStoleMyBread** [comment]
> You should have a look at [Kobra 2](https://github.com/ANYCUBIC-3D/Kobra2) as this here is intended for Kobra 3 related topics.

Yes but Anycubic has done the same with the 2 series, said they would release the code and never did violating the open source License.
